src/types.ts

```diff
@@ -4459,3 +4459,29 @@ export type EventHook = {
   created_at?: string;
   updated_at?: string;
 };
+
+export type UpdateChannelsBatchRequest = {
+  operation: string;
+  filter: UpdateChannelsBatchFilters;
+  members?: string[] | Array<NewMemberPayload>;
+  data?: Partial<ChannelData>;
+};
+
+export type UpdateChannelsBatchFilters = QueryFilters<{
+  cids?:
+    | RequireOnlyOne<Pick<QueryFilter<string>, '$in'>>
+    | RequireOnlyOne<Pick<QueryFilter<string[]>, '$eq'>>
```

Comment:
```suggestion
    | RequireOnlyOne<Pick<QueryFilter<string>, '$in' | '$eq'>>
```

Maybe could be simplified to this if we want to allow `$in` or `$eq` exclusively.